readme.md

```diff
@@ -123,6 +123,9 @@ Use [create-ink-app](https://github.com/vadimdemedes/create-ink-app) to quickly
 $ mkdir my-ink-cli
 $ cd my-ink-cli
 $ npx create-ink-app
+
+# Or create with TypeScript React
+$ npx create-ink-app --typescript
```

Comment:
Can you replace this into a separate code example?

Alternatively, create a TypeScript project:

```
$ npx create-ink-app --typescript
```

Reply:
Done.